src/transformers/models/vivit/modeling_vivit.py

```diff
@@ -104,9 +104,10 @@ def __init__(self, config):
             torch.zeros(1, self.patch_embeddings.num_patches + 1, config.hidden_size)
         )
         self.dropout = nn.Dropout(config.hidden_dropout_prob)
+        self.patch_size = config.tubelet_size
```

Comment:
It would be better to set this correctly to a patch_size i.e. a tuple of len 2, rather than assign it the tublet size

```suggestion
        self.patch_size = config.tubelet_size[1:]
```

Reply:
pushed the changes :+1: